py/server/tests/test_arrow.py

```diff
@@ -106,13 +106,21 @@ def test_arrow_types_unsigned_integers(self):
     def test_arrow_types_time(self):
         pa_types = [
             pa.time64('ns'),
-            pa.date32(),
-            pa.timestamp('ns', tz='MST'),
```

Comment:
It looks like that we don't support pa.timestamp(ns) with a tz, what about just no tz or tz=utc? If we flatly don't support pa.timestamp, maybe we should remove the 'java.time.Instant' mapping?

Reply:
Actually the test might be passing now. I suspect the test isn't looking at the time zones. I do think that the data is correct; the timezone is just like a display that applies to all elements. Instant's aren't in a particular time zone in general. Locally I only get lfs and postgres failures; so... we'll find out soon.